packages/jest-jasmine2/src/index.ts

```diff
@@ -137,7 +135,8 @@ async function jasmine2(
     });
   }
 
-  const snapshotState: SnapshotState = runtime
+  // TODO: make just snapshotState: SnapshotState when `jest-snapshot` is ESM
```

Comment:
You should be able to import directly now, thanks to #7973

Reply:
I have added `SnapshotStateType` in `jest-snapshot`. Otherwise I could not solve

Reply:
@jeysal ideas?